action.yml

```diff
@@ -64,4 +61,3 @@ runs:
         file_glob: true
         file: "${{ inputs.filename }}?(.asc)"
         tag: ${{ github.ref }}
-        overwrite: true
```

Comment:
Why the removal of this option?

Reply:
Good catch, ok I removed this hook which is currently not usable apparently:

```
    hooks:
      - id: check-github-actions
```

We can add it later, let's go baby steps it was not there initially.

Reply:
Added here c1747fc6976954d626fa50c84bed06910a4b7211